src/js/Mixins/Snapping.js

```diff
@@ -1,6 +1,7 @@
 const SnapMixin = {
     _initSnappableMarkers() {
         this.options.snapDistance = this.options.snapDistance || 30;
+        this.options.snapMiddle = this.options.snapMiddle;
```

Comment:
is this necessary? The line above was only to create a default of 30 🤔

Reply:
Removed